### PR TITLE
go.mod: update runc to latest git tip

### PR DESCRIPTION
**Pull request by kolyshkin**
This is in preparation to the upcoming rc93 release, to make sure
nothing is broken.

Brought to you by

	go get github.com/opencontainers/runc@master
	make tidy

rc93 is going to be released later this month (Nov 2020).

### CODE REVIEW

**Comment by bobbypage** [comment]
Sounds good -- only thing next week I'll be cutting a new cAdvisor release (v0.38) to be included in kubernetes 1.20. Since runc rc93  isn't released yet as you mentioned, I'm thinking to merge this after the release will be cut to ensure v0.38 will use stable version of runc.

File: resctrl/collector.go [hunk 1]
```diff
@@ -26,19 +26,14 @@ import (
 )
 
 type collector struct {
-	resctrl intelrdt.IntelRdtManager
+	resctrl intelrdt.Manager
 	stats.NoopDestroy
 }
 
 func newCollector(id string, resctrlPath string) *collector {
 	collector := &collector{
```
Comment (bobbypage) [inline resctrl/collector.go:34]: what do the changes here have to do with the runc update?

Comment (Creatone) [reply resctrl/collector.go:34]: @xiaochenshen made a new way of creating IntelRdt manager

https://github.com/opencontainers/runc/commit/933c4d31a77ac8b85982f960a09b7257fff51044
https://github.com/opencontainers/runc/commit/2c004a101ecd8b3b5604e53782ac1eb56859e4e5

**Comment by bobbypage** [comment]
Hi @kolyshkin, can you please rebase? Latest `v0.38` cAdvisor has been cut per https://github.com/google/cadvisor/pull/2712#issuecomment-723363109, so we can proceed here :)

File: resctrl/collector.go (hunk 1 above)
Comment (kolyshkin) [reply resctrl/collector.go:34]: Sorry for the late reply -- the reason for the change was duly explained in the commit message.

**Comment by kolyshkin** [comment]
rebased on top of current master

**Comment by bobbypage** [comment]
Looks like CI is failing:

```
2020-11-25T18:13:38.8867230Z go tidy changes are needed; please run make tidy
2020-11-25T18:13:38.8876154Z ##[error]make: *** [Makefile:95: presubmit] Error 1
2020-11-25T18:13:38.8886416Z go.mod diff:
2020-11-25T18:13:38.8887740Z --- /tmp/tmp.vpzSPjwT2Z	2020-11-25 18:13:38.329867298 +0000
2020-11-25T18:13:38.8888433Z +++ go.mod	2020-11-25 18:13:38.869895301 +0000
2020-11-25T18:13:38.8888946Z @@ -17,12 +17,9 @@
2020-11-25T18:13:38.8889544Z  	github.com/docker/go-units v0.4.0
2020-11-25T18:13:38.8890382Z  	github.com/euank/go-kmsg-parser v2.0.0+incompatible
2020-11-25T18:13:38.8891059Z  	github.com/gogo/protobuf v1.3.1
2020-11-25T18:13:38.8891807Z -	github.com/golang/protobuf v1.4.3 // indirect
2020-11-25T18:13:38.8892595Z -	github.com/google/go-cmp v0.5.2 // indirect
2020-11-25T18:13:38.8895203Z  	github.com/google/uuid v1.1.2 // indirect
2020-11-25T18:13:38.8895829Z  	github.com/gorilla/mux v1.8.0 // indirect
2020-11-25T18:13:38.8896404Z  	github.com/karrick/godirwalk v1.16.1
2020-11-25T18:13:38.8897618Z -	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
2020-11-25T18:13:38.8898364Z  	github.com/kr/pretty v0.2.0 // indirect
2020-11-25T18:13:38.8899402Z  	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
2020-11-25T18:13:38.8900226Z  	github.com/mindprince/gonvml v0.0.0-20190828220739-9ebdce4bb989
2020-11-25T18:13:38.8900634Z go.sum diff:
2020-11-25T18:13:38.8901156Z --- /tmp/tmp.sJZ9U0MGsj	2020-11-25 18:13:38.333867506 +0000
2020-11-25T18:13:38.8901649Z +++ go.sum	2020-11-25 18:13:38.869895301 +0000
2020-11-25T18:13:38.8902029Z @@ -50,13 +50,9 @@
2020-11-25T18:13:38.8902747Z  github.com/chzyer/logex v1.1.10/go.mod h1:+Ywpsq7O8HXn0nuIou7OrIPyXbp3wmkHB+jjWRnGsAI=
2020-11-25T18:13:38.8904171Z  github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e/go.mod h1:nSuG5e5PlCu98SY8svDHJxuZscDgtXS6KTTbou5AhLI=
2020-11-25T18:13:38.8905571Z  github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1/go.mod h1:Q3SI9o4m/ZMnBNeIyt5eFwwo7qiLfzFZmjNmxjkiQlU=
2020-11-25T18:13:38.8906846Z -github.com/cilium/ebpf v0.0.0-20200702112145-1c8d4c9ef775 h1:cHzBGGVew0ezFsq2grfy2RsB8hO/eNyBgOLHBCqfR1U=
2020-11-25T18:13:38.8908176Z -github.com/cilium/ebpf v0.0.0-20200702112145-1c8d4c9ef775/go.mod h1:7cR51M8ViRLIdUjrmSXlK9pkrsDlLHbO8jiB8X8JnOc=
2020-11-25T18:13:38.8909352Z  github.com/cilium/ebpf v0.2.0 h1:Fv93L3KKckEcEHR3oApXVzyBTDA8WAm6VXhPE00N3f8=
2020-11-25T18:13:38.8910291Z  github.com/cilium/ebpf v0.2.0/go.mod h1:To2CFviqOWL/M0gIMsvSMlqe7em/l1ALkX1PyjrX2Qs=
2020-11-25T18:13:38.8911285Z  github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
2020-11-25T18:13:38.8912617Z -github.com/containerd/console v1.0.0 h1:fU3UuQapBs+zLJu82NhR11Rif1ny2zfMMAyPJzSN5tQ=
2020-11-25T18:13:38.8913943Z -github.com/containerd/console v1.0.0/go.mod h1:8Pf4gM6VEbTNRIT26AyyU7hxdQU3MvAvxVI0sc00XBE=
2020-11-25T18:13:38.8915297Z  github.com/containerd/console v1.0.1 h1:u7SFAJyRqWcG6ogaMAx3KjSTy1e3hT9QxqX7Jco7dRc=
2020-11-25T18:13:38.8916424Z  github.com/containerd/console v1.0.1/go.mod h1:XUsP6YE/mKtz6bxc+I8UiKKTP04qjQL4qcS3XoQ5xkw=
2020-11-25T18:13:38.8917435Z  github.com/containerd/containerd v1.4.1 h1:pASeJT3R3YyVn+94qEPk0SnU1OQ20Jd/T+SPKy9xehY=
2020-11-25T18:13:38.8918211Z @@ -176,8 +172,6 @@
2020-11-25T18:13:38.8918818Z  github.com/kisielk/gotool v1.0.0/go.mod h1:XhKaO+MFFWcvkIS/tQcRk01m1F5IRFswLeQ+oQHNcck=
2020-11-25T18:13:38.8920028Z  github.com/konsorten/go-windows-terminal-sequences v1.0.1/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
2020-11-25T18:13:38.8921358Z  github.com/konsorten/go-windows-terminal-sequences v1.0.2/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
2020-11-25T18:13:38.8922888Z -github.com/konsorten/go-windows-terminal-sequences v1.0.3 h1:CE8S1cTafDpPvMhIxNJKvHsGVBgn1xWYf1NbHQhywc8=
2020-11-25T18:13:38.8924444Z -github.com/konsorten/go-windows-terminal-sequences v1.0.3/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
2020-11-25T18:13:38.8925637Z  github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515/go.mod h1:+0opPa2QZZtGFBFZlji/RkVcI2GknAs/DXo4wKdlNEc=
2020-11-25T18:13:38.8926626Z  github.com/kr/pretty v0.1.0 h1:L/CwN0zerZDmRFUapSPitk6f+Q3+0za1rQkzVuMiMFI=
2020-11-25T18:13:38.8927725Z  github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORNo=
2020-11-25T18:13:38.8928513Z @@ -193,8 +187,6 @@
2020-11-25T18:13:38.8929496Z  github.com/mindprince/gonvml v0.0.0-20190828220739-9ebdce4bb989/go.mod h1:2eu9pRWp8mo84xCg6KswZ+USQHjwgRhNp06sozOdsTY=
2020-11-25T18:13:38.8930826Z  github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible h1:aKW/4cBs+yK6gpqU3K/oIwk9Q/XICqd3zOX/UFuvqmk=
2020-11-25T18:13:38.8932105Z  github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible/go.mod h1:8AuVvqP/mXw1px98n46wfvcGfQ4ci2FwoAjKYxuo3Z4=
2020-11-25T18:13:38.8934136Z -github.com/moby/sys/mountinfo v0.1.3 h1:KIrhRO14+AkwKvG/g2yIpNMOUVZ02xNhOw8KY1WsLOI=
2020-11-25T18:13:38.8935807Z -github.com/moby/sys/mountinfo v0.1.3/go.mod h1:w2t2Avltqx8vE7gX5l+QiBKxODu2TX0+Syr3h52Tw4o=
2020-11-25T18:13:38.8937357Z  github.com/moby/sys/mountinfo v0.4.0 h1:1KInV3Huv18akCu58V7lzNlt+jFmqlu1EaErnEHE/VM=
2020-11-25T18:13:38.8938706Z  github.com/moby/sys/mountinfo v0.4.0/go.mod h1:rEr8tzG/lsIZHBtN/JjGG+LMYx9eXgW2JI+6q0qou+A=
2020-11-25T18:13:38.8940611Z  github.com/modern-go/concurrent v0.0.0-20180228061459-e0a39a4cb421/go.mod h1:6dJC0mAP4ikYIbvyc7fijjWJddQyLn8Ig3JB5CqoB9Q=
2020-11-25T18:13:38.8941893Z @@ -203,8 +195,6 @@
2020-11-25T18:13:38.8943246Z  github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
2020-11-25T18:13:38.8945050Z  github.com/morikuni/aec v1.0.0 h1:nP9CBfwrvYnBRgY6qfDQkygYDmYwOilePFkwzv4dU8A=
2020-11-25T18:13:38.8946763Z  github.com/morikuni/aec v1.0.0/go.mod h1:BbKIizmSmc5MMPqRYbxO4ZU0S0+P200+tUnFx7PXmsc=
2020-11-25T18:13:38.8948708Z -github.com/mrunalp/fileutils v0.0.0-20200520151820-abd8a0e76976 h1:aZQToFSLH8ejFeSkTc3r3L4dPImcj7Ib/KgmkQqbGGg=
2020-11-25T18:13:38.8950768Z -github.com/mrunalp/fileutils v0.0.0-20200520151820-abd8a0e76976/go.mod h1:x8F1gnqOkIEiO4rqoeEEEqQbo7HjGMTvyoq3gej4iT0=
2020-11-25T18:13:38.8952546Z  github.com/mrunalp/fileutils v0.5.0 h1:NKzVxiH7eSk+OQ4M+ZYW1K6h27RUV3MI6NUTsHhU6Z4=
2020-11-25T18:13:38.8954046Z  github.com/mrunalp/fileutils v0.5.0/go.mod h1:M1WthSahJixYnrXQl/DFQuteStB1weuxD2QJNHXfbSQ=
2020-11-25T18:13:38.8955890Z  github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRWi+5nqEBWmkhHvq77mSJWrCKwh8bxhgT7d/eI7P4U=
2020-11-25T18:13:38.8956954Z @@ -212,12 +202,8 @@
2020-11-25T18:13:38.8958249Z  github.com/opencontainers/go-digest v1.0.0/go.mod h1:0JzlMkj0TRzQZfJkVvzbP0HBR3IKzErnv2BNG4W4MAM=
2020-11-25T18:13:38.8959917Z  github.com/opencontainers/image-spec v1.0.1 h1:JMemWkRwHx4Zj+fVxWoMCFm/8sYGGrUVojFA6h/TRcI=
2020-11-25T18:13:38.8961220Z  github.com/opencontainers/image-spec v1.0.1/go.mod h1:BtxoFyWECRxE4U/7sNtV5W15zMzWCbyJoFRP3s7yZA0=
2020-11-25T18:13:38.8962441Z -github.com/opencontainers/runc v1.0.0-rc92 h1:+IczUKCRzDzFDnw99O/PAqrcBBCoRp9xN3cB1SYSNS4=
2020-11-25T18:13:38.8963727Z -github.com/opencontainers/runc v1.0.0-rc92/go.mod h1:X1zlU4p7wOlX4+WRCz+hvlRv8phdL7UqbYD+vQwNMmE=
2020-11-25T18:13:38.8965229Z  github.com/opencontainers/runc v1.0.0-rc92.0.20201124050251-06b737bb156b h1:D1hjnswpcHgsMMGjWL4sA13UDp6cSj+8jh6QX+HBo5Q=
2020-11-25T18:13:38.8967465Z  github.com/opencontainers/runc v1.0.0-rc92.0.20201124050251-06b737bb156b/go.mod h1:/jJp/o4HsLxTHyUjJ6GBqKpJDBU9HpsH27KhZK4Fh1s=
2020-11-25T18:13:38.8969494Z -github.com/opencontainers/runtime-spec v1.0.3-0.20200728170252-4d89ac9fbff6 h1:NhsM2gc769rVWDqJvapK37r+7+CBXI8xHhnfnt8uQsg=
2020-11-25T18:13:38.8971384Z -github.com/opencontainers/runtime-spec v1.0.3-0.20200728170252-4d89ac9fbff6/go.mod h1:jwyrGlmzljRJv/Fgzds9SsS/C5hL+LL3ko9hs6T5lQ0=
2020-11-25T18:13:38.8974005Z  github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d h1:pNa8metDkwZjb9g4T8s+krQ+HRgZAkqnXml+wNir/+s=
2020-11-25T18:13:38.8975905Z  github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d/go.mod h1:jwyrGlmzljRJv/Fgzds9SsS/C5hL+LL3ko9hs6T5lQ0=
2020-11-25T18:13:38.8977309Z  github.com/opencontainers/selinux v1.6.0 h1:+bIAS/Za3q5FTwWym4fTB0vObnfCf3G/NC7K6Jx62mY=
2020-11-25T18:13:38.8977979Z @@ -261,8 +247,6 @@
2020-11-25T18:13:38.8978703Z  github.com/sirupsen/logrus v1.4.1 h1:GL2rEmy6nsikmW0r8opw9JIRScdMF5hA8cOYLH7In1k=
2020-11-25T18:13:38.8979706Z  github.com/sirupsen/logrus v1.4.1/go.mod h1:ni0Sbl8bgC9z8RoU9G6nDWqqs/fq4eDPysMBDgk/93Q=
2020-11-25T18:13:38.8980763Z  github.com/sirupsen/logrus v1.4.2/go.mod h1:tLMulIdttU9McNUspp0xgXVQah82FyeX6MwdIuYE2rE=
2020-11-25T18:13:38.8982145Z -github.com/sirupsen/logrus v1.6.0 h1:UBcNElsrwanuuMsnGSlYmtmgbb23qDR5dG+6X6Oo89I=
2020-11-25T18:13:38.8983463Z -github.com/sirupsen/logrus v1.6.0/go.mod h1:7uNnSEd1DgxDLC74fIahvMZmmYsHGZGEOFrfsX/uA88=
2020-11-25T18:13:38.8984546Z  github.com/sirupsen/logrus v1.7.0 h1:ShrD1U9pZB12TX0cVy0DtePoCH97K8EtX+mg7ZARUtM=
2020-11-25T18:13:38.8985724Z  github.com/sirupsen/logrus v1.7.0/go.mod h1:yWOB1SBYBC5VeMP7gHvWumXLIWorT60ONWic61uBYv0=
2020-11-25T18:13:38.8986706Z  github.com/spf13/afero v1.2.2 h1:5jhuqJyZCZf2JRofRvN/nIFgIWNzPa3/Vz8mYylgbWc=
2020-11-25T18:13:38.8987306Z @@ -276,8 +260,6 @@
2020-11-25T18:13:38.8988086Z  github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81PSLYec5m4=
2020-11-25T18:13:38.8989134Z  github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd0=
2020-11-25T18:13:38.8990040Z  github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
2020-11-25T18:13:38.8991343Z -github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 h1:b6uOv7YOFK0TYG7HtkIgExQo+2RdLuwRft63jn2HWj8=
2020-11-25T18:13:38.8992739Z -github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2/go.mod h1:hkRG7XYTFWNJGYcbNJQlaLq0fg1yr4J4t/NcTQtrfww=
2020-11-25T18:13:38.8994175Z  github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635 h1:kdXcSzyDtseVEc4yCz2qF8ZrQvIDBJLl4S1c3GCXmoI=
2020-11-25T18:13:38.8995604Z  github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635/go.mod h1:hkRG7XYTFWNJGYcbNJQlaLq0fg1yr4J4t/NcTQtrfww=
2020-11-25T18:13:38.8996751Z  github.com/urfave/cli v1.22.1 h1:+mkCCcOFKPnCmVYVcURKps1Xe+3zP90gSYGNfRkjoIY=
2020-11-25T18:13:38.8997751Z @@ -385,7 +367,6 @@
2020-11-25T18:13:38.8998865Z  golang.org/x/sys v0.0.0-20191001151750-bb3f8db39f24/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9000537Z  golang.org/x/sys v0.0.0-20191026070338-33540a1f6037/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9002533Z  golang.org/x/sys v0.0.0-20191115151921-52ab43148777/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9004439Z -golang.org/x/sys v0.0.0-20191120155948-bd437916bb0e/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9006322Z  golang.org/x/sys v0.0.0-20191204072324-ce4227a45e2e/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9008342Z  golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9010276Z  golang.org/x/sys v0.0.0-20200106162015-b016eb3dc98e/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9011478Z @@ -400,7 +381,6 @@
2020-11-25T18:13:38.9013099Z  golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9014583Z  golang.org/x/sys v0.0.0-20200615200032-f1bc736245b1 h1:ogLJMz+qpzav7lGMh10LMvAkM/fAoGlaiiHYiFYdm80=
2020-11-25T18:13:38.9016052Z  golang.org/x/sys v0.0.0-20200615200032-f1bc736245b1/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9018069Z -golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9019959Z  golang.org/x/sys v0.0.0-20200909081042-eff7692f9009/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9021863Z  golang.org/x/sys v0.0.0-20200916030750-2334cc1a136f/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9023768Z  golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
2020-11-25T18:13:38.9029752Z ##[error]Process completed with exit code 2.
```

@kolyshkin Looks like it needs a `make tidy`

**Comment by bobbypage** [comment]
Closing due to runc already being updated in https://github.com/google/cadvisor/pull/2790

Please re-open if it makes sense to update to new runc or rc93 when it will be cut.

**Pull request closed without merging**